docs/navigation-parameters-reference.md

```diff
@@ -605,6 +605,18 @@ runTimeErrorHandler: {
   - A static **pathSegment** of value `settings` results in `example.com/settings`.
   - A dynamic **pathSegment** is prefixed with a colon and can load any value. Find out more about dynamic paths in Luigi [here](navigation-advanced.md#dynamically-changeable-paths).
 
+### showBreadcrumbs
+- - **type**: boolean
+- **description**: shows a breadcrumbs header for the node. See the [advanced navigation](navigation-advanced.md#breadcrumbs) document for more information.
```

Comment:
if breadcrumbs configuration is setup, the breadcrumbs show for all nodes. This property allows disabling the breadcrumb for one particular node by setting it to `false` for that node.
```suggestion
- **description**: if breadcrumbs configuration is setup, the breadcrumbs show for all nodes. This property allows disabling the breadcrumb for one particular node by setting it to `false` for that node.. See the [advanced navigation](navigation-advanced.md#breadcrumbs) document for more information.
```